### PR TITLE
Doc changes

### PR DESCRIPTION
**Pull request by shongesabbe**
I edited some of the docs found in the repo.
I recommend further refining them to change things to reflect better what you want out of contributors.
The link that I added on the install instructions for Windows I recommend is added to the other build docs. I don't have access to other operating systems to test if the installer on the website works for them, so I didn't add the link.

Also, the README.md mentioned an Android app that could run games, but I was not able to find a discreet link that isn't just the releases. 

Feel free to let me know if you have questions.

### CODE REVIEW

**Comment by ivan-mogilko** [comment]
Hello.
The idea of moving "Contribution" section into a separate is a fine one, and this is something that we should've done a long time ago.

In regards to "Code of conduct" part, to be frank I find its contents an excessive formalism. Maybe that sort of thing is meaningful for big communities, but our project so far was worked on by a very few people. If I had to write one, I'd make a simple paragraph asking to stay civil, be respectful, and that maintainers hold the right to remove posts or ban users who disrupt other people's participation in our project.
@ericoporto reminded me there was a proposal https://github.com/adventuregamestudio/ags/issues/885. I was not able to address that ticket back then (I recall having personal issues at that time). We could have used a concise form similar to the example there.

After this is merged, this way or another, we would also have to adjust some information, because it may be not in sync in the recent situation (there have been few deviation from initial plan, I neglected updating the readme for a while).

**Comment by ericoporto** [comment]
I agree with @ivan-mogilko words, I think that part could be removed from this PR and left to be added later. 

I think you can then also squash all these commits to a single one - if you use the GitHub Desktop app, this can be easily done by selecting all your added commits, right clicking to squash, and then just adjusting the commit message to describe what is changed, keeping the commit title small - the GitHub desktop app should show a lamp icon if you go beyond the recommended size of the commit title, which I think is around 50 -60 characters. Then you can force-push to the branch which you did the pr - I don’t remember if the GitHub Desktop app has force push in the interface, but it’s simply typing `git push —force` in the command line.

**Comment by ivan-mogilko** [comment]
I checked the proposed changes more carefully now, and I cannot agree with completely moving the "Branches and releases" section from README.md to CONTRIBUTING.md. This information about branches may be essential not only to potential contributors, but to any person who would like to use this repository. I believe it should stay in README.md; but it seems necessary to also duplicate this info in CONTRIBUTING.md for convenience.

Other things seem okay overall. There are things that I'd rephrase or fix formatting in, but that may be done after.

**Comment by ivan-mogilko** [comment]
@shongesabbe I was not sure if you have an intent to adjust this PR, and having no response in more than a week decided to push the selected changes from your pr with certain modification. I've mentioned you in the commit info: 274ab97958c839afd9dabcf144f6ebf5aedb9ad7

I'll close this pr now.
